azuki-core/src/main/kotlin/com/anaplan/engineering/azuki/core/runner/VerifiableScenarioRunner.kt

```diff
@@ -0,0 +1,95 @@
+package com.anaplan.engineering.azuki.core.runner
+
+import com.anaplan.engineering.azuki.core.scenario.VerifiableScenario
+import com.anaplan.engineering.azuki.core.system.*
+import org.slf4j.LoggerFactory
+
+class VerifiableScenarioRunner<S : VerifiableScenario<AF, CF>, AF : ActionFactory, CF : CheckFactory>(
```

Comment:
I don't know what could go in an interface.. certainly the run method would be different

We could make the result generic, but it would feel like you're just doing it for the sake of it

Reply:
The run method would definitely do different things, but would its usage actually be different?

Reply:
discussed & agreed to maintain